//package core.io.protocol;

message DseHeartbeat 
{
	optional int32  		time    = 1;
}
